include/openssl/lhash.h.in

```diff
@@ -165,6 +174,24 @@ void OPENSSL_LH_node_usage_stats_bio(const OPENSSL_LHASH *lh, BIO *out);
     } \
     LHASH_OF(type)
 
+# ifndef OPENSSL_NO_DEPRECATED_3_1
+#  define DEFINE_LHASH_OF_DEPRECATED(type) \
+    static ossl_unused ossl_inline OSSL_DEPRECATEDIN_3_1 void lh_##type##_node_stats_bio(const LHASH_OF(type) *lh, BIO *out) \
```

Comment:
That's a very long line...  We still have an 80 column requirement in the code style.
I suggest resolving that by doing what the GNU folks do, a line break before the function name:

``` C
    static ossl_unused ossl_inline OSSL_DEPRECATEDIN_3_1 void \
    lh_##type##_node_stats_bio(const LHASH_OF(type) *lh, BIO *out) \
```

Reply:
A lot of these inlines already exceeded the length limit, but I guess we may as well clean them up while we're here.